app/controllers/poems_controller.rb

```diff
@@ -4,17 +4,17 @@ def new
   end
 
   def create
-    @poem = Poem.create(poem_params)
-
+    @poem = current_user.poems.create(poem_params)
+    
     respond_with @poem
   end
 
   def show
-    @poem = Poem.find(params[:id])
+    @poem = Poem.find(params[:id]) || Poem.new
```

Comment:
Also, it's probably confusing to a user who attempts to view an existing poem and instead sees a new poem (presumably with a bunch of blank fields).